alcs-api/src/user/user.controller.ts

```diff
@@ -13,8 +13,6 @@ export class UserController {
   constructor(private userService: UserService) {}
 
   @Get()
-  @UseGuards(RoleGuard)
-  @UserRoles(AUTH_ROLE.ADMIN)
```

Comment:
Why are you removing the guard here?

Reply:
thanks, a leftover from test, I'll bring it back